pkg/ddevapp/config_test.go

```diff
@@ -194,6 +194,56 @@ func TestConfigCommand(t *testing.T) {
 	}
 }
 
+// TestConfigCommandDocrootDetection asserts the default docroot is detected.
+func TestConfigCommandDocrootDetection(t *testing.T) {
+	// Set up tests and give ourselves a working directory.
+	assert := asrt.New(t)
+
+	testMatrix := []string{"web", "docroot", "htdocs", "_www", "public"}
```

Comment:
As above, these should be defined outside of the code so both test and application can use the same values in the same order.